lib/commands/context.js

```diff
@@ -299,13 +299,13 @@ extensions.onPageChange = async function (pageChangeNotification) {
       let needsPageLoad = (() => {
         let item = (arr) => {
           return _.filter(arr, (obj) => {
-            return obj.id === this.curContext;
```

Comment:
I think it can theoretically happen that this.curContext has a certain value and then this.contexts changes during runtime (for example, because getContexts() is called and the context from this.curContext is not available anymore). If this happens, item(this.contexts) would return undefined and a pageLoad would only be triggered if contextArray contains this.curContext. However, I don't know if this is necessary as I don't know the code well enough.